No elastic-local image at hub.docker.com

I'm trying to build use this provider for the first time, but `make build` fails with the error `pull access denied for elastic-local, repository does not exist or may require 'docker login'`. However, I'm logged into hub.docker.com, is there a step missing in the README?

Repo steps:
```
$ mkdir -p $GOPATH/src/github.com/ewilde
$ cd $GOPATH/src/github.com/ewilde
$ git clone git@github.com:ewilde/terraform-provider-kibana
Cloning into 'terraform-provider-kibana'...
remote: Enumerating objects: 2445, done.
remote: Total 2445 (delta 0), reused 0 (delta 0), pack-reused 2445
Receiving objects: 100% (2445/2445), 3.35 MiB | 3.68 MiB/s, done.
Resolving deltas: 100% (462/462), done.
$ cd terraform-provider-kibana/
$ docker login
Authenticating with existing credentials...
Login Succeeded
$ make build
==> Checking that code complies with gofmt requirements...
go vet .
building docker ELK_VERSION: KIBANA_TYPE: MAIN_VERSION:
TF_ACC=1 go test -v ./kibana -run "TestAcc"
2019/06/17 20:01:04 Could not start kibana: : API error (404): {"message":"pull access denied for elastic-local, repository does not exist or may require 'docker login'"}
FAIL    github.com/ewilde/terraform-provider-kibana/kibana      2.297s
make: *** [test] Error 1
```

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -2,9 +2,18 @@
 TEST_PATH ?= "TestAcc"
 GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 $(eval REMAINDER := $$$(ELK_VERSION))
-MAIN_VERSION := $(shell echo $(ELK_VERSION) | head -c 3)
 
 default: build test
+
+check-elk-version:
+ifndef ELK_VERSION
+	$(error ELK_VERSION is undefined)
+endif
+
+check-kibana-type:
+ifndef KIBANA_TYPE
+	$(error KIBANA_TYPE is undefined)
+endif
 
 travisbuild: deps default
 
@@ -17,7 +26,9 @@ build: fmtcheck vet test
 	@cp $(GOPATH)/bin/terraform-provider-kibana ~/.terraform.d/plugins/terraform-provider-kibana
 	@echo "Build succeeded"
 
-docker-build:
+docker-build: check-elk-version check-kibana-type
+	$(eval MAIN_VERSION := $(shell echo $(ELK_VERSION) | head -c 3))
+
 	@echo building docker ELK_VERSION:$(ELK_VERSION) KIBANA_TYPE: $(KIBANA_TYPE) MAIN_VERSION: $(MAIN_VERSION)
 	@if [ "$(ELK_VERSION)" = "./..." ]; then \
 		echo "ERROR: Set ELK_VERSION to a specific version. For example,"; \

--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ Enter the provider directory and build the provider
 
 ```sh
 $ cd $GOPATH/src/github.com/ewilde/terraform-provider-kibana
-$ make build
+$ ELK_VERSION=6.2.3 KIBANA_TYPE=KibanaTypeVanilla make build
 ```
 
 Using the provider
@@ -202,19 +202,19 @@ If you wish to work on the provider, you'll first need [Go](http://www.golang.or
 To compile the provider, run `make build`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
 
 ```sh
-$ make bin
+$ ELK_VERSION=6.2.3 KIBANA_TYPE=KibanaTypeVanilla make build
 ...
 $ $GOPATH/bin/terraform-provider-kibana
 ...
 ```
 
-In order to test the provider, you can simply run `make test`.
+In order to test the provider, you can simply run `make test`. Note that `ELK_VERSION` and `KIBANA_TYPE` are used to control the test targets. The full list of test targets is visible in `.travis.yml`.
 
 ```sh
-$ make test
+$ ELK_VERSION=6.2.3 KIBANA_TYPE=KibanaTypeVanilla make test
 ```
 
-In order to run the full suite of Acceptance tests, run `make testacc ELK_VERSION=6.2.1 KIBANA_TYPE=KibanaTypeVanilla`.
+In order to run the full suite of Acceptance tests, run `ELK_VERSION=6.2.3 KIBANA_TYPE=KibanaTypeVanilla make testacc`.
 
 *Note:* Acceptance tests create real resources, and often cost money to run.
 

